auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt

```diff
@@ -114,7 +114,8 @@ class PhoneAuthActivity : Activity() {
         val optionsBuilder = PhoneAuthOptions.newBuilder(auth)
             .setPhoneNumber(phoneNumber)       // Phone number to verify
             .setTimeout(60L, TimeUnit.SECONDS) // Timeout and unit
-            .setActivity(this)                 // Activity (for callback binding)
+            .setActivity(this)                 // Optional  Activity (for callback binding)
+                                               // If no activity is passed, recaptcha verification can not be done.
```

Comment:
Same suggestion as above
```suggestion
            .setActivity(this)                 // (optional) Activity for callback binding
                                               // If no activity is passed, reCAPTCHA verification can not be used.
```